<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleDCUPage</class>
 <widget class="QWidget" name="VehicleDCUPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:black;
}
QLabel
{
	font: 20px, &quot;微软雅黑&quot;;
	color:white;
background-color:blue;
}
QPushButton
{
	background-color:brown ;
	border-top: 2px solid white;	
	border-left: 2px solid white;
}</string>
  </property>
  <widget class="QLabel" name="MOTORIM2lbl">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>390</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="KMM2lbl">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>250</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_9">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>125</y>
     <width>120</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PANTOMP1_1lbl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>电机电流(A)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>牵引逆变器</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VDCM2lbl">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="HSBCM2lbl">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>110</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VDCMP2lbl">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VDCM1lbl">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="KMMP1lbl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MOTORIMP1lbl">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>390</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>自动降弓</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="KMM1lbl">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>250</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_8">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>2</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="HSBCMP2lbl">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>110</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MOTORIMP2lbl">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>390</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VDCMP1lbl">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>分离接触器</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="KMMP2lbl">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>250</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MOTORIM1lbl">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>390</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PANTOMP2_1lbl">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>40</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="HSBCMP1lbl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>电压(Vdc)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="HSBCM1lbl">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>高速断路器</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>牵引/电制动力(KN)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_15">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>70</y>
     <width>2</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_10">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>120</y>
     <width>2</width>
     <height>350</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_11">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>120</y>
     <width>2</width>
     <height>350</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_12">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>125</y>
     <width>120</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TCUEBMP1lbl">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>460</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TCUEBM2lbl">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>460</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TCUEBMP2lbl">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>460</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TCUEBM1lbl">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>460</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TCUIVM2lbl">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>320</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TCUIVM1lbl">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TCUIVMP2lbl">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>320</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TCUIVMP1lbl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>lbl1_14</zorder>
  <zorder>lbl1_11</zorder>
  <zorder>lbl1_8</zorder>
  <zorder>lbl1_10</zorder>
  <zorder>lbl1_15</zorder>
  <zorder>lbl1_12</zorder>
  <zorder>lbl1_9</zorder>
  <zorder>MOTORIM2lbl</zorder>
  <zorder>KMM2lbl</zorder>
  <zorder>PANTOMP1_1lbl</zorder>
  <zorder>lbl1_6</zorder>
  <zorder>lbl1_5</zorder>
  <zorder>VDCM2lbl</zorder>
  <zorder>HSBCM2lbl</zorder>
  <zorder>VDCMP2lbl</zorder>
  <zorder>VDCM1lbl</zorder>
  <zorder>KMMP1lbl</zorder>
  <zorder>MOTORIMP1lbl</zorder>
  <zorder>lbl1</zorder>
  <zorder>KMM1lbl</zorder>
  <zorder>HSBCMP2lbl</zorder>
  <zorder>MOTORIMP2lbl</zorder>
  <zorder>VDCMP1lbl</zorder>
  <zorder>lbl1_4</zorder>
  <zorder>KMMP2lbl</zorder>
  <zorder>MOTORIM1lbl</zorder>
  <zorder>PANTOMP2_1lbl</zorder>
  <zorder>HSBCMP1lbl</zorder>
  <zorder>lbl1_3</zorder>
  <zorder>HSBCM1lbl</zorder>
  <zorder>lbl1_2</zorder>
  <zorder>TCUEBMP1lbl</zorder>
  <zorder>TCUEBM2lbl</zorder>
  <zorder>TCUEBMP2lbl</zorder>
  <zorder>TCUEBM1lbl</zorder>
  <zorder>TCUIVM2lbl</zorder>
  <zorder>TCUIVM1lbl</zorder>
  <zorder>TCUIVMP2lbl</zorder>
  <zorder>TCUIVMP1lbl</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
